pkgs/applications/system/howdy/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv
+, lib
+, bzip2
+, fetchFromGitHub
+, fetchurl
+, fmt
+, gettext
+, inih
+, installShellFiles
+, libevdev
+, meson
+, ninja
+, pam
+, pkg-config
+, python3
+} @ args:
+let
+  data = import ./sources.nix args;
```

Comment:
What is the point of having sources in a separate file?

Reply:
No point anymore. In past versions I've used it to separate pam_howdy and howdy. I will move those into `default.nix`.